src/video/font.cpp

```diff
@@ -843,7 +843,8 @@ void CFont::MeasureWidths()
 		while (sp < gp) {
 			// Some accented glyphes are not perfectly aligned on the glyph grid (like ï or î ).
 			// So we need to do -1 to not compute width on the next glyph.
-			const unsigned char *lp = sp + (G->Width - 1); 
+			//const unsigned char *lp = sp + (G->Width - 1); 
```

Comment:
This is actually needed for french and german to work on my machine, to you have problems with this?

Reply:
I reverted the complete commit and now german (including display of umlauts) works and no crash. I didn't check each single change in this commit. Should I enable some single lines to test it? (Hint: I didn't test if 'ß' works. Need to find an intro with this letter.

Reply:
Ah sorry and in the pullrequest are both, LUA and the german font issue. My bad! Should I reject and change this pullrequest?

Reply:
I could confirm 'ß' not working. I'll close and open a new one